packages/react-select/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
you should definitely re-run the migration script I mentioned earlier since you're not using the new tsconfig setup here